### PR TITLE
Add aggregation:capi_infrastructure_crd_versions metric to Grafana Cloud

### PR DESCRIPTION
**Pull request by mnitchev**
Towards: https://github.com/giantswarm/roadmap/issues/3344

Exposes the installed CAPI infrastructure CRDs installed on MCs

### Checklist

- [x] Update CHANGELOG.md
- [ ] Add [Unit tests](https://github.com/giantswarm/prometheus-rules/#testing)
- [ ] Follow [Alert structure](https://github.com/giantswarm/prometheus-rules/#how-alerts-are-structured)
- [ ] Consider [creating a dashboard](https://docs.giantswarm.io/getting-started/observability/grafana/custom-dashboards/) ([guidelines](https://intranet.giantswarm.io/docs/product/ux/guidelines/dashboards/)) (if it does not exist already) to help oncallers monitor the status of the issue.
- [ ] Request review from oncall area, as well as team (e.g: `oncall-kaas-cloud` GitHub group).

### CODE REVIEW

**Review by calvix — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Add `aggregation:capi_infrastructure_crd_versions` metric to Cortex.
```
Comment (QuentinBisson) [inline CHANGELOG.md:12]:
```suggestion
- Add `aggregation:capi_infrastructure_crd_versions` metric to Grafana Cloud.
```

File: helm/prometheus-rules/templates/recording-rules/grafana-cloud.rules.yml [hunk 2]
```diff
@@ -550,3 +550,7 @@ spec:
     rules:
       - expr: sum(trivy_compliance_info{title!~".*Pod.*"}) by (cluster_id, cluster_type, customer, installation, pipeline, provider, region, compliance_id, compliance_name, severity, title, status)
         record: aggregation:cluster_compliance_info
+  - name: capi_crds.grafana-cloud.recording
```
Comment (QuentinBisson) [inline helm/prometheus-rules/templates/recording-rules/grafana-cloud.rules.yml:553]: Not blocking but could you document why you are adding this?

**Review by QuentinBisson — APPROVED** [review]

File: helm/prometheus-rules/templates/recording-rules/grafana-cloud.rules.yml (hunk 2 above)
Comment (mnitchev) [reply helm/prometheus-rules/templates/recording-rules/grafana-cloud.rules.yml]: I added a small comment but it seems kind of redundant. We just want to keep track of what crd versions are installed :shrug:

**Comment by mnitchev** [comment]
CI is broken but it looks like it was broken by 4.0.0 and not the thing I added

**Pull request merged**